.github/ISSUE_TEMPLATE.md

```diff
@@ -4,13 +4,13 @@
 - [ ] Update your protoc to the [latest version](https://github.com/google/protobuf/releases).
 - [ ] Update your copy of the `grpc-gateway` library to the latest version from github:
   ```sh
-  go get -u github.com/grpc-ecosystem/grpc-gateway
+  go get -u github.com/grpc-ecosystem/grpc-gateway/v2
   ```
 - [ ] Delete the `protoc-gen-grpc-gateway` and `protoc-gen-openapiv2` binary from your `PATH`,
   and reinstall the latest versions:
   ```sh
-  go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
-  go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-openapiv2
+  go get -u github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway
+  go get -u github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2
```

Comment:
Could we remove the `-u` from these too? Sorry for being so picky, this is really great.

Reply:
As this is the instructions to update binaries to the latest version, shouldn't we keep the `-u` flag?

> The -u flag instructs get to use the network to update the named packages
and their dependencies. By default, get uses the network to check out
missing packages but does not use it to look for updates to existing packages.

Reply:
Fair point, but I'd rather we did `go get github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway@latest` than use the older flag. I don't think it means what people think it means for modules.

Reply:
Ok, done. I've used `@latest` only for `github.com/grpc-ecosystem/grpc-gateway/v2`, afterwards, it should not be necessary when re-installing the binaries.

And I've just seen that... this file is not used anymore! It it is a legacy GitHub issue template, and the new templates, in `.github/ISSUE_TEMPLATE` directory, are used instead. I guess both should be merged.

Reply:
Doh! Feel free to follow up with another PR.